torch/distributions/bernoulli.py

```diff
@@ -46,6 +46,19 @@ def __init__(self, probs=None, logits=None, validate_args=None):
             batch_shape = self._param.size()
         super(Bernoulli, self).__init__(batch_shape, validate_args=validate_args)
 
+    def expand(self, batch_shape):
+        batch_shape = torch.Size(batch_shape)
+        new = self.__new__(Bernoulli)
+        if 'probs' in self.__dict__:
+            new.probs = self.probs.expand(batch_shape)
+            new._param = new.probs
+        else:
+            new.logits = self.logits.expand(batch_shape)
+            new._param = new.logits
+        super(Bernoulli, new).__init__(batch_shape, validate_args=False)
```

Comment:
Why can't `validate_args` be `self.validate_args`? And is `validate_args` actually required? Because, if `validate_args` is true in the parent distribution, then the arguments are validated there, and this is merely an expanded version. Sorry if I missed something.